boot/bootutil/src/bootutil_priv.h

```diff
@@ -180,15 +180,17 @@ _Static_assert(sizeof(boot_img_magic) == BOOT_MAGIC_SZ, "Invalid size for image
                  (hdr)->ih_ver.iv_minor,                                  \
                  (hdr)->ih_ver.iv_revision,                               \
                  (hdr)->ih_ver.iv_build_num)
-
+#if defined(MCUBOOT_SWAP_USING_MOVE)
 #if MCUBOOT_SWAP_USING_MOVE
```

Comment:
Can we not just change line 184 to the if defined, instead of wrapping with an additional #if?

Reply:
Yes, it is better to do that. I have changed line 184 to [#if defined(MCUBOOT_SWAP_USING_MOVE) && MCUBOOT_SWAP_USING_MOVE].
Because someone may set MCUBOOT_SWAP_USING_MOVE to 0.